ppr-ui/src/components/search/SearchBarList.vue

```diff
@@ -138,12 +138,15 @@ export default defineComponent({
         }
       }),
       isSingleSearchOption: computed((): boolean => {
-        return (hasPprEnabled.value && !hasMhrEnabled.value) || (!hasPprEnabled.value && hasMhrEnabled.value)
+        return ((hasPprEnabled.value && !hasMhrEnabled.value) || (!hasPprEnabled.value && hasMhrEnabled.value)) &&
```

Comment:
Looks like we'll be using these combo flags a lot, maybe we can combine them in a new PR.

Reply:
Definitely!